priv/templates/phx.gen.live/live_helpers.ex

```diff
@@ -4,6 +4,29 @@ defmodule <%= inspect context.web_module %>.LiveHelpers do
 
   alias Phoenix.LiveView.JS
 
+  @doc """
+  Renders a flash message.
+
+  The rendered flash receives a `:type` that will be used to define
+  proper classes to the element, and a `:message` which will be the
+  inner HTML, if any exists.
+
+  ## Examples
+
+      <.flash type="info" message="User created" />
+  """
+  def flash(assigns) do
+    message = Map.get(assigns.flash, assigns.kind)
+
+    if is_binary(message) do
```

Comment:
```suggestion
    if message = Map.get(assigns.flash, assigns.kind) do
```
Currently, phoenix supports anything that implements `Phoenix.HTML.Safe` as flash messages. While I agree that displaying `nil` is not the best idea, the function component should probably support iolists, live components or anything else.

Reply:
if the key is there, we should use it, so we can ignore the type check entirely